cluster/gce/gci/configure-helper-common.sh

```diff
@@ -1430,7 +1430,112 @@ scrape_configs:
     - targets: ['scale_out_proxy_ip:8404']
 EOF
   sed -i -e "s/scale_out_proxy_ip/$PROXY_RESERVED_IP/g" /tmp/prometheus-metrics.yaml
-  ./prometheus --config.file="/tmp/prometheus-metrics.yaml" --web.listen-address=":9090" --web.enable-admin-api &
+  nohup ./prometheus --config.file="/tmp/prometheus-metrics.yaml" --web.listen-address=":9090" --web.enable-admin-api > prometheus.log 2>&1 &
```

Comment:
@pdgetrf Please review this. I don't know if the config generated in Sonya's case will have the scale_out_proxy_ip placeholder.

Reply:
this looks good

Reply:
do we need nohup here?

Reply:
we have a dedicated log file to record the cluster start up process and any issues: /var/log/master-init.log. This prometheus installation will generate too many logs information for itself. so use nohup and redirect its install logs to different files.

Reply:
got it. for some reason I don't see the resolve button here. consider this comment resolved.